docs/whatsnew/1.7.0/1_7_0.rst

```diff
@@ -0,0 +1,47 @@
+Pylake 1.7.0
+============
+
+.. only:: html
+
+Here is an overview of features that were released in Pylake `v1.7.0`.
+
+Improved force calibration
+--------------------------
+
+Pylake 1.7.0 comes with several improvements to the force calibration workflow, making it easier to work with force calibration data.
+It is now possible to recalibrate a force slice using a force calibration item::
+
+  recalibrated_force1x = f.force1x.recalibrate_force(new_calibration)
+
+Force calibration items now have properties containing the raw calibration data in the properties :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.voltage`, :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.driving` and :attr:`~lumicks.pylake.calibration.ForceCalibrationItem.sum_voltage`.
+Recalibrating a force slice with a single calibration setting overridden becomes a simple three lines::
```

Comment:
I would have expected an example below that includes `voltage`, `driving`, or `sum_voltage`. Wouldn’t it make sense to first mention the example with a single calibration item overwritten, and then mention the properties?

Reply:
Do you think that sounds okay?

Reply:
Fixing, thanks!